docs/src/browsers.md

```diff
@@ -115,18 +306,20 @@ pwsh bin/Debug/netX/playwright.ps1 install msedge
 mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="install msedge"
 ```
 
-Run with `--help` option to see full list of the browsers that can be installed this way.
 
 :::warning
-Google Chrome or Microsoft Edge installations will not be isolated. They will be installed at the
-default global location that depends on your operating system.
+Google Chrome or Microsoft Edge installations will be installed at the
+default global location of your operating system overriding your current browser installation.
 :::
 
-### When to use Google Chrome & Microsoft Edge and when not to?
+Run with `--help` option to see full list of the browsers that can be installed this way.
+
+
+#### When to use Google Chrome & Microsoft Edge and when not to?
 
 **Defaults**
```

Comment:
This bold renders exactly the same as previous header. Perhaps make it a bullet list?

Reply:
changed to smaller headings. bullets didnt look good